bindings/node/index.d.ts

```diff
@@ -207,6 +207,9 @@ export interface ClientEncryptionOptions {
 
   /**
    * Optional callback to override KMS providers per-context.
+   *
+   * @deprecated Installing optional dependencies will automatically refresh kms
```

Comment:
https://jira.mongodb.org/browse/NODE-4731 created to remove this in the next major release, deprecate for now.